extensions/nba-game-viewer/package.json

```
@@ -18,43 +18,67 @@
     {
       "name": "upcoming-games",
       "title": "View Upcoming Games",
-      "subtitle": "NBA",
-      "description": "Fetches upcoming NBA games",
+      "subtitle": "NBA & WNBA",
+      "description": "Fetches upcoming NBA or WNBA games",
       "mode": "view"
     },
     {
       "name": "standings",
       "title": "View Standings",
-      "subtitle": "NBA",
-      "description": "Fetches NBA standings",
+      "subtitle": "NBA & WNBA",
+      "description": "Fetches NBA or WNBA standings",
       "mode": "view"
     },
     {
       "name": "headlines",
       "title": "View Headlines",
-      "subtitle": "NBA",
-      "description": "Fetches NBA league headlines from ESPN",
+      "subtitle": "NBA & WNBA",
+      "description": "Fetches NBA or WNBA league headlines from ESPN",
       "mode": "view"
     }
   ],
-  "preferences": [{
-    "name": "conferences",
-    "description": "Choose Standings default conference",
-    "type": "dropdown",
-    "title": "Conferences",
-    "data": [{
-        "title": "Eastern Conference",
-        "value": "Eastern"
-      }, {
-        "title": "Western Conference",
-        "value": "Western"
-      }, {
-        "title": "League",
-        "value": "League"
-    }],
-    "required": true,
-    "default": "Eastern"
-  }],
+  "preferences": [
+    {
+      "name": "league",
+      "description": "Choose NBA or WNBA League",
+      "type": "dropdown",
+      "title": "League",
+      "data": [
+        {
+          "title": "NBA",
+          "value": "nba"
+        },
+        {
+          "title": "WNBA",
+          "value": "wnba"
+        }
+      ],
+      "required": true,
+      "default": "NBA"
+    },
+    {
+      "name": "conferences",
```

Usually the name for a dropdown should be singular

```suggestion
      "name": "conference",
```